src/modules/add.js

```diff
@@ -37,12 +42,14 @@ export default class AddMovies {
     let sum = list.summary;
     sum = sum.replace(/^"(.*)"$/, '$1');
     description.innerHTML = `${sum}`;
-    likesP.innerHTML = `<i class="fa-regular fa-heart"></i>likes <span>${count}</span>`;
+    likeText.innerText = 'likes';
     button.href = list.show;
     button.innerText = 'comments';
 
     // appending elements with each other
```

Comment:
hello cluab, its amazing how you sdded functionality of the likes and the styling of the heart icon is stunning.
good job.